util/collections/machine_collection.go

```diff
@@ -107,7 +118,7 @@ func (s Machines) Difference(machines Machines) Machines {
 
 // SortedByCreationTimestamp returns the machines sorted by creation timestamp.
 func (s Machines) SortedByCreationTimestamp() []*clusterv1.Machine {
```

Comment:
I was wondering if we should make the SortedByCreationTimestamp private as well, as it is being used in other public methods.

Reply:
I see. Thanks :)